python_modules/libraries/dagster-airbyte/dagster_airbyte_tests/experimental/test_asset_specs.py

```diff
@@ -46,3 +60,105 @@ def test_translator_spec(
         # Test the asset key for the connection table
         the_asset_key = next(iter(all_assets_keys))
         assert the_asset_key.path == ["test_prefix_test_stream"]
+
+        first_asset_metadata = next(asset.metadata for asset in all_assets)
+        assert AirbyteMetadataSet.extract(first_asset_metadata).connection_id == TEST_CONNECTION_ID
+
+
+def test_cached_load_spec_single_resource(
+    fetch_workspace_data_api_mocks: responses.RequestsMock,
+) -> None:
+    with environ(
+        {"AIRBYTE_CLIENT_ID": TEST_CLIENT_ID, "AIRBYTE_CLIENT_SECRET": TEST_CLIENT_SECRET}
+    ):
+        workspace = AirbyteCloudWorkspace(
+            workspace_id=TEST_WORKSPACE_ID,
+            client_id=EnvVar("AIRBYTE_CLIENT_ID"),
+            client_secret=EnvVar("AIRBYTE_CLIENT_SECRET"),
+        )
+
+        # load asset specs a first time
```

Comment:
weird grammar here

Reply:
Update in [e13a65e](https://github.com/dagster-io/dagster/pull/26432/commits/e13a65e2516d66b024335b4f304534b304e09fd6)